MediaBrowser.Providers/TV/TheTVDB/TvDbClientManager.cs

```diff
@@ -180,21 +191,23 @@ public Task<string> GetEpisodeTvdbId(EpisodeInfo searchInfo, CancellationToken c
                 episodeQuery.FirstAired = searchInfo.PremiereDate.Value.ToString("yyyy-MM-dd");
             }
 
-            return GetEpisodeTvdbId(Convert.ToInt32(seriesTvdbId), episodeQuery, cancellationToken);
+            return GetEpisodeTvdbId(Convert.ToInt32(seriesTvdbId), episodeQuery, cancellationToken, language);
         }
 
-        public async Task<string> GetEpisodeTvdbId(int seriesTvdbId, EpisodeQuery episodeQuery, CancellationToken cancellationToken)
+        public async Task<string> GetEpisodeTvdbId(int seriesTvdbId, EpisodeQuery episodeQuery,
```

Comment:
why this function is special? all others accept `language` _before_ `cancellationToken`